.env.default

```diff
@@ -1,2 +1,8 @@
+### APP ########################################################################
 DATABASE_URL="postgres://Dandoh:dandoh@127.0.0.1:5432/webhaskell?sslmode=disable"
 JWT_SECRET="my_jwt_secret"
+
+### PostgreSQL #################################################################
```

Comment:
Perhaps we should add comments indicating that these variables are for `docker` only.